test/pop_protocol_test.erl

```diff
@@ -134,10 +134,14 @@ basic_test() ->
     ?assertError(_, lists:foldl(FoldFn, PD0, [E2])),
     ?assertError(_, lists:foldl(FoldFn, PD0, [B1, E3])),
 
-    %% Error for when same verifier adds two different blocks
+    %% same verifier adds two different blocks
     T = make_transaction(PrivateKey, PublicKey, 0, hype_chain),
     B1T = make_block(genesis, 1, PrivateKey, PublicKey, 1, 110, [T]),
-    ?assertError(_, lists:foldl(FoldFn, PD0, [B1, B1T])), 
+    PDT = lists:foldl(FoldFn, PD0, [B1, B1T]),
+
+    MinHash = min(maps:get(this_id, B1), maps:get(this_id, B1T)),
+    HeadHash = maps:get(this_id, PDT#protocol_data.head_block),
+    ?assertEqual(MinHash, HeadHash),
```

Comment:
why are these equal?

Reply:
this is the test for one verifier submitting two blocks. one with smallest hash is the head

Reply:
:o I see